akka-docs-dev/rst/scala/stream-flows-and-basics.rst

```diff
@@ -24,12 +24,12 @@ Element
   downstream. Buffer sizes are always expressed as number of elements independently form the actual size of the elements.
 Back-pressure
   A means of flow-control, a way for consumers of data to notify a producer about their current availability, effectively
-  slowing down the upstream source to match their consumption speeds.
-  In the context of Akka Streams back-pressure is always understood as *non-blocking* and *asynchronous*
+  slowing down the upstream producer to match their consumption speeds.
```

Comment:
There are cases where we could not avoid saying producer, but that does not mean that we should not prefer source everywhere. If you can rephrase the above sentence to avoid producer there, too, then it is even better.